docs/source/contributing.md

```diff
@@ -2,154 +2,88 @@
 
 ## Prerequisites
 
-To develop on scylla-operator, your environment must have the following:
-
-1. [Go 1.13](https://golang.org/dl/)
-    * Make sure [GOPATH](https://github.com/golang/go/wiki/SettingGOPATH) is set to `GOPATH=$HOME/go`.
-2. [Kustomize v3.1.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.1.0)
-3. [kubebuilder v2.3.1](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v2.3.1)
-4. [Docker](https://docs.docker.com/install/)
```

Comment:
some make targets are still using docker and kustomize is still used in some of the scripts,  did you remove these on purpose?